pyam/iiasa.py

```diff
@@ -66,14 +66,49 @@ def auth(self):
     @lru_cache()
     def metadata(self):
         """
-        Metadata (e.g., models, scenarios, etc.) in the connected data
+        Metadata (e.g., models, scenarios, etc.) of the connected data
```

Comment:
I'd mention "assigned categories, quantitative summary indicators" as examples of metadata rather than "models, scenarios" (because those are already part of the `data` IamDataFrame).

Reply:
Ah, but this is different. This is the metadata of the *connected data source*. That includes `runId`, `model`,  etc.

Reply:
got it - but then I would call this function `scenario_list()` similar to the `ixmp` API.